src/components/chat/chat-thread.tsx

```diff
@@ -23,16 +22,15 @@ interface IProps {
   focusDocument?: string;
   focusTileId?: string;
   isDocumentView?: boolean;
-  setIsDocumentView?: Dispatch<SetStateAction<boolean>>;
+  handleDocView?: () => void;
```

Comment:
It looks like this isn't used, so it should be removed.

Reply:
Done